src/sauce/settings.js

```diff
@@ -10,12 +10,13 @@ var clc = require("cli-color");
 /*eslint-disable no-magic-numbers*/
 var config = {
   // required:
-  tunnelId: process.env.SAUCE_TUNNEL_ID,
+  // tunnelId: process.env.SAUCE_TUNNEL_ID,
```

Comment:
Let's just delete instead of commenting out.  Git history keeps the memory around.